snooty/postprocess.py

```diff
@@ -2037,23 +2037,26 @@ def __init__(self, context: Context) -> None:
         super().__init__(context, "wayfinding", {"include", "sharedinclude"})
 
 
-class MethodSelectorHandler(NestedDirectiveHandler):
+class MethodSelectorHandler(Handler):
     """Handles page-level validations for method-selector directive and its children."""
 
     def __init__(self, context: Context) -> None:
-        super().__init__(context, "method-selector", {"include", "sharedinclude"})
+        super().__init__(context)
+        self.method_selector_name = "method-selector"
```

Comment:
If this is only read in one place, I would say either make it a global constant or just use the literal directly on line 2075: instance members like this are quite slow to access, and generally less unnecessarily mutable state is better.